drivers/clock_control/clock_control_nrf2_fll16m.c

```diff
@@ -28,21 +28,13 @@ BUILD_ASSERT(DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) == 1,
 
 #define FLL16M_HAS_LFXO DT_NODE_HAS_STATUS_OKAY(FLL16M_LFXO_NODE)
 
-#define FLL16M_LFXO_ACCURACY DT_PROP(FLL16M_LFXO_NODE, accuracy_ppm)
 #define FLL16M_HFXO_ACCURACY DT_PROP(FLL16M_HFXO_NODE, accuracy_ppm)
```

Comment:
And this one

Reply:
I don't see any accuracy setting for HFXO in BICR

Reply:
Right. My mistake. So the accuracy should stay here.